src/benchmarklib/benchmark_runner.cpp

```diff
@@ -41,6 +43,17 @@ BenchmarkRunner::BenchmarkRunner(const BenchmarkConfig& config, std::unique_ptr<
     const auto scheduler = std::make_shared<NodeQueueScheduler>();
     CurrentScheduler::set(scheduler);
   }
+
+  if (config.validate) {
+    // If we wanted to, we could probably implement this, but right now, it does not seem to be worth the effort
+    Assert(!config.enable_scheduler, "Cannot use validation with enabled scheduler");
+    Assert(config.clients == 1, "Cannot use validation with multiple clients");
```

Comment:
This line is a bit redundant, because if the scheduler is disabled then the "clients" config doesn't have any effect, since everything is executed sequentially anyway.